### PR TITLE
Add some e-kara cartridges + added Hi-kara

### PR DESCRIPTION
**Pull request by mamehaze**
new machines marked as NOT_WORKING
-----------------------------------------------
Hi-Kara  [David Haywood, Team Europe]

new software list items
-------------------------
ekara:
Enkamichi Volume 9 (Japan) (EC0064-ENK) [David Haywood, Team Europe]
Hibari Misora PR-06 (Japan) [David Haywood, Team Europe]
TV Pop Junior Volume 3 (Japan) (GC0009-TPJ) [David Haywood, Team Europe]
Kids Song Mini Volume 2 (Japan) (MC0008-KSM) [David Haywood, Team Europe]
Super Cartridge: Best Hit (Japan) (SP-01) [David Haywood, Team Europe]
evio:
Concerto Selection [David Haywood, Team Europe]
hikara:
Vol 1: J-Pop 1 [David Haywood, Team Europe]
Vol 2: J-Pop 2 [David Haywood, Team Europe]
Vol 4: Anime 1 [David Haywood, Team Europe]
Hi-kara 5-song capacity user cartridge (used, with 5 songs) [David Haywood, Team Europe]

------------------------
(GC0009 and SP-01 are playable with Popira / Popira 2, SP-01 is also playable with Taiko de Popira, all run with eKara but with no microphone input)

The Hi-Kara emulation can play the cartridges, but there's no microphone input.

------------------------

this also confirms sc-8 as a good dump, it has been dumped from a 2nd cartridge with idential results, and the same 'crash on score screen' issue on real hardware.  It is unknown if a fixed version exists (it seems like a serious bug, and they did do a cart revision of another cart just to remove the WTC image so it would be strange if there isn't a fixed version)

------------------------

Also removed an old workaround for excite fishing that is no longer needed with the improved SEEPROM emulation from last year.

### CODE REVIEW

**Comment by mamehaze** [comment]
pcb
pcb
pcb
pcb
pcb
pcb

**Comment by mamehaze** [comment]
image
pcb
pcb
pcb
pcb_back
xtal1
u7
xtal2
dumping-rig
pcb_front

File: hash/hikara.xml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+		<publisher>Takara Tomy</publisher>
+		<part name="cart" interface="ekara_cart">
+			<dataarea name="rom" size="0x100000">
+				<rom name="A29L800ATV.u1" size="0x100000" crc="679f92e9" sha1="9af880f7f09c0e616d0f0b3cd499ab42de3ab210"/>
+				<!-- temp patch to boot -->
+				<rom size="1" value="0xf0" offset="0x8ec4" loadflag="fill"/>
+				<rom size="1" value="0xd0" offset="0x8ec9" loadflag="fill"/>
+			</dataarea>
+		</part>
+	</software>
```
Comment (cuavas) [inline hash/hikara.xml:57]: The four tracks visible in the track listing screenshot are:
* 奇跡の香りダンス。 – 松浦亜弥 (Kiseki no Kaori Dance. – Matsūra Aya)
* さくらんぼ – 大塚愛 (Sakuranbo – Ōtsuka Ai)
* 愛のために。 – 上戸彩 (Ai no Tame ni. – Ueto Aya)
* DOUBLE – BoA (Double BoA)

The screenshots above and below the track listing show 真赤な太陽 (Makka no Taiyō), DOUBLE and YUME日和 (Yume Biyori), respectively.  With no descriptions for the screenshots, it’s difficult to work out what they’re supposed to go with.  The title card screenshots also don’t show artist/performer names.  Is one of them the fifth track on this cartridge?  What performer does it show on the track listing if it is?

I’ve added track listing comments for cartridges in ekara_japan_web.xml with user content.

File: hash/ekara_japan.xml [hunk 2]
```diff
@@ -923,7 +923,17 @@ license:CC0-1.0
...
+		<description>Enkamichi Volume 9 (Japan) (EC0064-ENK)</description>
+		<year>2001</year>
+		<publisher>Takara</publisher>
+		<info name="alt_title" value="演歌道ベスト Vol.9" />
+		<part name="cart" interface="ekara_cart">
+			<dataarea name="rom" size="0x100000">
+				<rom name="ec0064-enk.u1" size="0x100000" crc="ec2fefb3" sha1="1a56f251d32ff3eb5a20c70ac040c6e0f7d9307d"/>
+			</dataarea>
+		</part>
+	</software>
```
[the remark this answers is not in the document]
Comment (mamehaze) [reply hash/ekara_japan.xml:936]: I don't speak, read or write Japanese.  I don't know.

File: hash/hikara.xml (hunk 1 above)
Comment (mamehaze) [reply hash/hikara.xml]: Well I'm sure you can add these, I don't speak, read or write Japanese.

File: hash/ekara_japan.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (cuavas) [reply hash/ekara_japan.xml:936]: From a quick search, volumes 2, 3, 5, 6 and 9 don’t seem to have “Best” in the title.  It’s most likely a copy/paste error.  Please remove “ベスト” from the Japanese titles of the Enkamichi cartridges in this list.  (The one in ekara_japan_p.xml is correct.)

File: hash/hikara.xml (hunk 1 above)
Comment (mamehaze) [reply hash/hikara.xml]: https://www.youtube.com/watch?v=IWe8zHDPHaU  is this one running.

Comment (cuavas) [reply hash/hikara.xml]: キューティーハニー – 倖田來未 (Cutie Honey – Kōda Kumi)
KissHug – aiko
さくらんぼ – 大塚愛 (Sakuranbo – Ōtsuka Ai)
Dream Fighter – Perfume
ポリリズム – Perfume (Polyrhythm – Perfume)

**Pull request merged**